src/path-graph.js

```diff
@@ -15,6 +15,8 @@ import { CollapsibleSection } from './collapsible-section.js';
 import { NumberBox } from './number-box.js';
 import { TextButton } from './buttons.js';
 import { downloadSvg } from './util.js';
+import { Tooltip } from './tooltip.js';
```

Comment:
This file is getting bigger and bigger. I wonder whether it makes sense to split it into multiple ones.

Reply:
I think we can create an issue to split some of these files up, because this is not the only one getting very big. It will take a bit of time, so it probably makes sense to do this in a different PR.

Reply:
Sounds a good plan to me.